### PR TITLE
changes hhhhh

### PR DESCRIPTION
**Pull request by militsa-radoyichich**
gagagagaga

### CODE REVIEW

File: trial project.R [hunk 1]
```diff
@@ -1,3 +1,6 @@
 x <- 1:100
 
-y <- cumsum(x)
\ No newline at end of file
+y <- cumsum(x)
+
+
+#hehehehhehehhe fuck
```
Comment (PavlovaAlexandra) [inline trial project.R:6]: No fuck, Milica, please @militsa-radoyichich

Comment (militsa-radoyichich) [reply trial project.R:6]: fuck you man

Comment (PavlovaAlexandra) [reply trial project.R:6]: ok man, approved

**Review by PavlovaAlexandra — APPROVED** [review]
Good job!

**Pull request merged**